### ISSUE TITLE
FileSaver.js causes requireJS errors in console

### ISSUE THREAD

**Issue by ping4tucker** [AllenFang/react-bootstrap-table#1147]
FileSaver.js is causing the following error:

```
Uncaught Error: Mismatched anonymous define() module: function () {
			return ExecutionEnvironment;
```

It looks like the current version of FileSaver.js being used was grabbed before this fix:
https://github.com/eligrey/FileSaver.js/commit/7b242c2d781c1d2b93a163cb5a137080693b387d

Without that fix, it causes the error above.

**Comment by AllenFang** [comment]
@ping4tucker, ok, I'll update it, thank and sorry for reply late.

**Comment by AllenFang** [comment]
Fixed on `v3.1.3`
